aten/src/ATen/native/Activation.cpp

```diff
@@ -394,7 +394,8 @@ TORCH_IMPL_FUNC(gelu_out_cpu) (
 ) {
 auto approximate_type = get_gelutype_enum(approximate);
 #if AT_MKLDNN_ENABLED()
-  if (use_mkldnn(self) && (approximate_type == GeluType::None)) {
+  // TODO: oneDNN's gelu fails to provide enough accuracy. Disable for now.
+  if (0 && use_mkldnn(self) && (approximate_type == GeluType::None)) {
```

Comment:
Do we expect this will be fixed in v3.1?

Reply:
Yes. The gelu issue should be fixed by oneDNN. I disabled mkldnn gelu here just to run CI checks. I will undo this once oneDNN's fix is available publicly.